src/main/resources/db/migration/V1_25__activity_migration.sql

```diff
@@ -0,0 +1,12 @@
+alter table activity_mapping drop column label;
+
+create table activity_migration_mapping (
+    nomis_course_activity_id       bigint       not null primary key,
+    activity_schedule_id           bigint       not null,
+    activity_schedule_id2          bigint,
+    label                          varchar(20)  not null,
+    when_created                   timestamp with time zone not null default now()
+);
```

Comment:
Decided that activity migration deserved its own table because:

- it has 2 additional columns used for the migration only
- the migration isn't linked to the sync in any way